pkg/auth/service.go

```diff
@@ -1902,13 +1903,44 @@ func NewAPIAuthService(apiEndpoint, token string, secretStore crypt.SecretStore,
 	} else {
 		cache = &DummyCache{}
 	}
-	res := &APIAuthService{
+	healthRes := &APIAuthService{
```

Comment:
I think this name is misleading, that's not a Health check response.

Reply:
Mistake - fixed